python/ray/rllib/models/tf/lstm_v1.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+
+from ray.rllib.models.model import Model
+from ray.rllib.models.tf.misc import linear, normc_initializer
+from ray.rllib.policy.rnn_sequencing import add_time_dimension
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils import try_import_tf
+
+tf = try_import_tf()
+
+
+# Deprecated: see as an alternative models/tf/recurrent_tf_modelv2.py
```

Comment:
why not log a warning?

Reply:
It's still used by default so it would be weird to log it.